package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
can we delete the outer package.json and lock file? it uses the one inside

Reply:
Yes, accidental addition, ty